frontend/lobby/team.tsx

```diff
@@ -42,11 +42,17 @@ interface Props extends OwnProps, DispatchProp<RootState> {
     otherTeamMembers: Array<OtherTeamMember>;
 }
 interface State {
+    teamView: boolean,
+    hasSeenSplash: boolean
```

Comment:
`hasSeenSplash` can be removed from here, and from the `constructor`. We can also remove the `splashDone()` method which I must have accidentally left in. There's no splash on the Team page.

Reply:
Done in the latest commit